sandbox/extract-dataset-metadata_emp.R

```diff
@@ -6,6 +6,7 @@
 
 library(stringr)
 library(tidyverse)
+library(sf)
 library(devtools)
 
 # Set working directory
```

Comment:
This should not be necessary if you use `datadir` as it was set up in the example (and how you are mostly using it!).

Reply:
theoretically yes.....but in practice this doesn't work for me (for mysterious unknown reasons). happy to remove these library calls though!

Reply:
I feel like we talked about this at some point and maybe you were having problems with packages external to ofo too?

Reply:
Oops I meant not the package loads, but the line after them that sets the working directory.